nasa/openmct
# 4588 - Removed summary widget creatability and updated composition policy

Closes #4588 

### All Submissions:


### Author Checklist


How to test:
1. Open the create menu
2. Ensure that `Summary Widget` is not available in the menu
